src/main/java/org/jenkins/tools/test/hook/AbstractMultiParentHook.java

```diff
@@ -128,13 +128,16 @@ protected void configureLocalCheckOut(UpdateSite.Plugin currentPlugin, File loca
     }
 
     /**
-     * Returns the folder where the multimodule project parent will be checked out
+     * Return the folder where the multi-module project will be checked out. This should be the name
+     * of the plugin's Git repository.
```

Comment:
This doesn't really matter one way or another, but to be consistent with plugins that are not multi-module, recommend the name of the Git repository.

Reply:
do we ever have a valid scenario where we have a multi module project with > 1 plugin where we would have different versions of those plugins to test in the release?
if that is the case does using the git repo name cause issues in testing (as you can only have one version checked out at once and then depending what you are running you may have a plugin from both the `war` and the reactor.